pallets/thea-executor/src/lib.rs

```diff
@@ -48,7 +48,7 @@ pub trait WeightInfo {
 pub mod pallet {
 	use super::*;
 	use frame_support::{
-		log,
+		__private::log,
```

Comment:
What is this `__private`? is this a mistake?

Reply:
@zktony Please update here.